shell/plugins/version.js

```diff
@@ -0,0 +1,10 @@
+import { setIsRancherPrime } from '@shell/config/version';
+export default async function({ $axios }) {
+  try {
+    const response = await $axios.get('/rancherversion');
```

Comment:
This request seems to trigger the schema request from the error in the E2E.
There must be some race condition issues which are totally unknown from me. Please check it out with @nwmac or @richard-cox as they implemented most of it and may have an idea to avoid it.

Reply:
Just to keep updated also the PR, it seems like the issue was in somehow produced also by the E2E, as it pointed to `v2.6-head` generating schema errors.